src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -149,6 +149,9 @@ the main body of the message as well as a quote.
 							<ActionButton
 								:close-after-click="true"
 								@click.stop="handleMarkAsUnread">
+								<template #icon>
+									<CheckCircleOutline size="20" />
```

Comment:
```suggestion
									<CheckCircleOutline
										decorative
										title=""
										:size="16" />
```

Not a too big fan of the icon, but I guess it clarifies the action and better than no icon